### PR TITLE
Bump Spark / Zeppelin to 1.5.2 / 0.5.6 tags

### PR DESCRIPTION
**Pull request by zmerlynn**
From https://github.com/kubernetes/application-images/pull/5

### CODE REVIEW

**Comment by zmerlynn** [comment]
@k8s-bot e2e test this, #21810, #21484

**Comment by zmerlynn** [comment]
@k8s-bot e2e test this: #21810

File: examples/spark/spark-gluster/spark-master-controller.yaml [hunk 1]
```diff
@@ -15,7 +15,7 @@ spec:
     spec:
       containers:
         - name: spark-master
-          image: gcr.io/google_containers/spark-master:1.5.1_v2
+          image: gcr.io/google_containers/spark-master:1.5.2_v1
```
Comment (mattf) [inline examples/spark/spark-gluster/spark-master-controller.yaml]: this needs to be changed to spark:1.5.2_v1 and the command: /start-master needs to be added

File: examples/spark/spark-gluster/spark-worker-controller.yaml [hunk 2]
```diff
@@ -16,7 +16,7 @@ spec:
     spec:
       containers:
         - name: spark-worker
-          image: gcr.io/google_containers/spark-worker:1.5.1_v2
+          image: gcr.io/google_containers/spark-worker:1.5.2_v1
```
Comment (mattf) [inline examples/spark/spark-gluster/spark-worker-controller.yaml]: this needs to be changed to spark:1.5.2_v1 and the command: /start-worker needs to be added

File: examples/spark/spark-gluster/spark-master-controller.yaml (hunk 1 above)
Comment (zmerlynn) [inline examples/spark/spark-gluster/spark-master-controller.yaml]: Wow, I just saw this. I have no idea how I missed your comment.

**Comment by zmerlynn** [comment]
Done. PTAL

**Comment by mattf** [comment]
LGTM

**Comment by eparis** [comment]
This PR has added the `cherrypick-candidate` label but did not set a milestone. This is an invalid state and will not be considered for cherrypick-ing to any release branch. The `cherrypick-candidate` label will be automatically removed from such PRs in the future.

**Comment by bgrant0607** [comment]
@zmerlynn Why would you like this to be cherrypicked?

**Comment by zmerlynn** [comment]
It would be nice if the Spark example actually went out closer to current
in the release, and the cherry pick itself is very low risk. (The images
were pushed in application-images a bit ago, and this was actually sitting
in my queue for a bit, I apologize. Matt doesn't have maintainer, but I saw
his LGTM and just hadn't checked on it in a while.)

**Comment by zmerlynn** [comment]
(Concretely, the upgrade to Spark 1.5.2 was "strongly recommended".)

**Pull request merged**